listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -1,28 +1,31 @@
-# coding=utf-8
+from datetime import datetime
+from time import time
 
 import ujson
-import redis
-from time import time
-from redis import Redis
 from typing import Optional
 
 from listenbrainz.listen import Listen
 from listenbrainz.listenstore import ListenStore
-from datetime import datetime
+from brainzutils import cache
+from listenbrainz.utils import create_path, init_cache
 
 
 class RedisListenStore(ListenStore):
 
-    RECENT_LISTENS_KEY = "lb_recent_sorted"
+    RECENT_LISTENS_KEY = "rl-"
     RECENT_LISTENS_MAX = 100
+    PLAYING_NOW_KEY = "pn."
     LISTEN_COUNT_PER_DAY_EXPIRY_TIME = 3 * 24 * 60 * 60  # 3 days in seconds
-    LISTEN_COUNT_PER_DAY_KEY_FORMAT = "lb_listen_count_for_day_{}"
+    LISTEN_COUNT_PER_DAY_KEY_FORMAT = "lc-day-"
 
 
     def __init__(self, log, conf):
         super(RedisListenStore, self).__init__(log)
-        self.redis = Redis(host=conf['REDIS_HOST'], port=conf['REDIS_PORT'], decode_responses=True)
-        self.ns = conf['REDIS_NAMESPACE']
+
+        # Initialize brainzutils cache
+        init_cache(host=conf['REDIS_HOST'], port=conf['REDIS_PORT'],
```

Comment:
Is this the only place where we init_cache? If possible we should only set this once in `create_app`, perhaps [after we create the redis connection](https://github.com/metabrainz/listenbrainz-server/blob/bbf6a832b30a7e3fa1b35f808299646d75719c32/listenbrainz/webserver/__init__.py#L97). By this point, `cache._r` should be set so that you can read it here.

Reply:
Sadly it isn't ready. Because of the way we have our imports calling create_redis() imports the redis_listenstore and then tries to use the cache object that is not ready. Let's fix this later if you feel strongly about it.

Reply:
this ties in with LB-879, which means that we can do it there. OK to leave here.